environment-api/src/main/java/com/sequenceiq/environment/api/v1/environment/model/response/EnvironmentLoadBalancerUpdateResponse.java

```diff
@@ -0,0 +1,71 @@
+package com.sequenceiq.environment.api.v1.environment.model.response;
+
+import java.util.Set;
+
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+
+import com.sequenceiq.common.api.type.PublicEndpointAccessGateway;
+import com.sequenceiq.environment.api.doc.environment.EnvironmentModelDescription;
+import com.sequenceiq.environment.api.v1.environment.model.base.LoadBalancerUpdateStatus;
+
+@ApiModel(value = "EnvironmentLoadBalancerUpdateResponse")
+public class EnvironmentLoadBalancerUpdateResponse {
+
+    public static final String NO_ERROR = "None";
+
+    @ApiModelProperty(EnvironmentModelDescription.PUBLIC_ENDPOINT_ACCESS_GATEWAY)
+    private PublicEndpointAccessGateway requestedPublicEndpointGateway = PublicEndpointAccessGateway.DISABLED;
+
+    @ApiModelProperty(EnvironmentModelDescription.ENDPOINT_ACCESS_GATEWAY_SUBNET_IDS)
+    private Set<String> requestedEndpointSubnetIds = Set.of();
+
+    @ApiModelProperty(EnvironmentModelDescription.LB_UPDATE_STATUS)
+    private LoadBalancerUpdateStatus status = LoadBalancerUpdateStatus.NOT_STARTED;
+
+    @ApiModelProperty(EnvironmentModelDescription.LB_UPDATE_FLOWID)
+    private String flowId;
```

Comment:
I know that's a normal pattern in the Stack service, but in `EnvironmentEndpoint` all the methods are either void or return a response object. That's the pattern I was following. But I'm okay with changing it to return a `FlowIdentifier` instead of that would be preferred.

Reply:
I meant replacing the `String flowId` with `FlowIdentifier`, not replacing the response object